### ISSUE TITLE
Feature request: keep track of pixel unit information

### ISSUE THREAD

**Issue by SoleneW** [biocompibens/SME#10]
Hi, 

I would like to know if you could add the pixel width and height properties in the output image of the fiji plugin (Image --> Properties...)? 

To explain my thought, my problem is that I have a 3d stack for which the properties are as follow for 1 pixel:
width = 0.07 micron 
height = 0.07 micron
depth = 0.15 micron

In the output 2D image, the unit of length is in pixel and not in micron. And for 1 pixel:
width = 1.00
height = 1.00
depth = 1.00

But I would like to keep the unit of length in micron. Would it be possible? 

Thanks, 
Solène

**Comment by erexhepa** [comment]
yes it should be possible. From the last time I was working with the plugin I saw that the ImagePlus allows you to fix the pixel size in the metadata. But meanwhile this can also be done post processing via a script or via one of the options of ImageJ. All these changes weather you do it prior or post projection will not affect the projection per se.
